### PR TITLE
C3 fattouh92 s16 view projects shared with me

### PR DESCRIPTION
**Pull request by abulazm22**
[Issue #210 ]
@SalmaFG 
@m0hamed 
@nohahesham 
@Mirna-Yacout 
Please review

### CODE REVIEW

**Comment by abulazm22** [comment]
Projects/index.html.erb
projects_controller.rb {def index}

**Comment by Mirna-Yacout** [comment]
UML reviewed

File: arability/app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -1,66 +1,541 @@
...
-  # params: 
-  #   none
-  # returns:
-  #   on success: returns an array of projects of the developer currently logged in.
-  #   on failure: notifies the user that he can't see this page.
+  #    Mohamed Tamer
+  # description:
+  #    function shows all the projects that a certain developer owns and the projects shared with him
+  # Params:
+  #    current_gamer: the current currently logged in, will be nil if there is no logged in gamer
```
Comment (salmafg) [inline arability/app/controllers/projects_controller.rb]: Sentence is missing word "gamer".

**Comment by m0hamed** [comment]
This cannot be merged into master until you have the latest changes from your teamates

**Pull request merged**